tests/unit/workflows/nodejs_npm_esbuild/test_workflow.py

```diff
@@ -269,8 +268,8 @@ def test_workflow_sets_up_esbuild_actions_with_download_dependencies_and_depende
 
         self.assertIsInstance(workflow.actions[0], CopySourceAction)
         self.assertIsInstance(workflow.actions[1], NodejsNpmInstallAction)
-        self.assertIsInstance(workflow.actions[2], CleanUpAction)
-        self.assertIsInstance(workflow.actions[3], EsbuildBundleAction)
+        self.assertIsInstance(workflow.actions[2], EsbuildBundleAction)
+        self.assertIsInstance(workflow.actions[3], CleanUpAction)
```

Comment:
the order did change, to simplify the code logic. There should be no impact in the end-result, however. We just need to clean up the dependencies dir before we move the updated dependencies in there, but the bundling itself happens on another folder.